models/nft/fantom/nft_fantom_transfers.sql

```diff
@@ -23,12 +23,11 @@ SELECT 'fantom'                                 AS blockchain
      , 'fantom' || t.evt_tx_hash || '-erc721-' || t.contract_address || '-' || t.tokenId || '-' || t.from || '-' ||
        t.to || '-' || '1' || '-' || t.evt_index AS unique_transfer_id
 FROM {{ source('erc721_fantom', 'evt_transfer') }} t
-    {% if is_incremental() %}
-    ANTI JOIN {{this}} anti_table
-    ON t.evt_tx_hash = anti_table.tx_hash
-    {% endif %}
 {% if is_incremental() %}
-WHERE t.evt_block_time >= date_trunc("day", now() - interval '1 week')
+    ANTI JOIN {{this}} anti_table
+        N t.evt_tx_hash = anti_table.tx_hash
```

Comment:
seems like a typo here

Reply:
woah, crazy, will fix now. Thanks ser @jeff-dude

Reply:
uhh weird can't find this on vs code lemme check properly, I don't have the **where** filter as well. 

checked GitHub and as well can't find that on line 26 etc